docs/reasons.rst

```diff
@@ -0,0 +1,120 @@
+.. _reasons:
+
+===============
+Why PyScaffold?
+===============
+
+|
+
+Stable and battle-proven
```

Comment:
Battle-proven or battle-tested? I only know the former but was is the difference?

Reply:
I would suppose they are synonyms. I have seen battle-proven in a few places before (like: https://pypi.org/project/pymultigen/, https://github.com/ltjax/ushiro, http://docs.viewflow.io), but it is true that battle-tested is more common. I should change